static/js/components/SearchTextbox.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import React from "react"
+
+type Props = {
+  onChange: Function,
+  onClear: Function,
+  onSubmit: Function,
+  value: string
+}
+
+const SearchTextbox = ({ onChange, onClear, onSubmit, value }: Props) => (
+  <div className="search-wrapper">
+    <i className="material-icons search-icon" onClick={onSubmit}>
+      search
+    </i>
+    <i className="material-icons clear-icon" onClick={onClear}>
+      clear
+    </i>
+    <input type="text" className="search" value={value} onChange={onChange} />
+  </div>
```

Comment:
I have a `<form onSubmit=...>` which is wrapping the element in my WIP PR for the search page. Should I do that in the storybook? Should it be part of this component?

Reply:
Ah, gotcha. It might be nice to have the 'Enter' UI work in storybook, but wrapping the thing in a form gets away from the component being responsible for implementing internally all of the UI behavior it's responsible for (since the UI behavior is actually coming from an external wrapping component). Given that it's implemented with a form in the UI I guess as it is is probably the best option.